parachain-template/runtime/src/lib.rs

```diff
@@ -732,6 +732,18 @@ impl_runtime_apis! {
 		}
 	}
 
+	#[cfg(feature = "try-runtime")]
```

Comment:
I don't think the template runtime has the `try-runtime` feature in it's Cargo.toml, yet